notebooks/IDC_segmentation_primer.ipynb

```diff
@@ -1,1927 +1 @@
-{
```

Comment:
<p>Change "bones contour" to "left lung contour"</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/ImagingDataCommons/IDC-Tutorials/pull/22/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='88'/>